### PR TITLE
Add serviceable attribute to projects.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

File: src/Microsoft.AspNet.DataProtection.Shared/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.AspNet.DataProtection.Shared/Properties/AssemblyInfo.cs]: I think this project isn't a real project, right? Build-time only.

Comment (NTaylorMullen) [inline src/Microsoft.AspNet.DataProtection.Shared/Properties/AssemblyInfo.cs]: :+1:

**Comment by NTaylorMullen** [comment]
Updated.

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**